### PR TITLE
Add plugin & puppet Blog

### LINKED ISSUE #264: Publish the blog/news for our Plugin Publishing Event
Link to https://github.com/wechaty/PMC/issues/10

Video Playback: <https://youtu.be/tfGZXoe_aA4>

### PR DESCRIPTION
**Pull request by rickyyin98**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [ ] Add Blog Header, including title, author, date.
e.g.

``` md
---
 title: "'Score Your Face Photo' a ML&Wechaty practice"
 date: 2017-09-18 09:00 +0800
 author: huyingxi
 ---
 > Author: [@huyingxi](https://github.com/huyingxi/wechaty_selfie) enjoying ML&Wechaty at BUPT
```

- [ ] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [ ] Embed the photo & video before publishing, save all external file to the blog `/download/2018` directory.
- [ ] Add `<!--more-->` section to just show the abstract in blog homepage.
- [ ] Add teaser image for the blog
- [ ] Add your self as an author to the author data file at <https://github.com/wechaty/wechaty.github.io/blob/master/_data/authors.yml> with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!

[See more](https://github.com/wechaty/wechaty.github.io#contribute-guideline)

### CODE REVIEW

**Comment by huan** [comment]
It seems that the content is this post is exact the same as the summer 2020?

Please check and fix, thank you very much.

**Comment by rickyyin98** [comment]
> It seems that the content is this post is exact the same as the summer 2020?
> 
> Please check and fix, thank you very much.

Fixed

**Comment by lijiarui** [comment]
I suggest adding 4 speakers ppt link in the blog.

File: _posts/2020-07-22-wechaty-plugin-milestone.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    padding-bottom:56.25%;
+    padding-top:30px;
+    height:0;
+    overflow:hidden;
+">
+<iframe width="560" height="315" src="https://www.youtube.com/embed/tfGZXoe_aA4" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
+![Wechaty Plugin活动](/assets/2020/wechaty-plugin-milestone/plugin.png)
+[Wechaty 作者 李卓桓 Wechaty项目和社区生态介绍](https://www.bilibili.com/video/BV1vA411v73c)
+[Wechaty Plugin 设计师，腾讯微信小程序框架开源项目Wepy 作者 Gcaufy 插件系统开发历程及应用](https://www.bilibili.com/video/BV1Et4y1y7Gc)
```
Comment (huan) [inline _posts/2020-07-22-wechaty-plugin-milestone.md:64]: Could you please change the agenda to a list for a better display?

like:

```md
- [Wechaty 作者 李卓桓 Wechaty项目和社区生态介绍](https://www.bilibili.com/video/BV1vA411v73c)
- [Wechaty Plugin 设计师，腾讯微信小程序框架开源项目Wepy 作者 Gcaufy 插件系统开发历程及应用](https://www.bilibili.com/video/BV1Et4y1y7Gc)
- [15:00 - 15:30 Wechaty Puppet Donut/Padplus 作者 高原 Wechaty 插件的商业化落地展望](https://www.bilibili.com/video/BV13T4y1J7Ta)
- [15:30 - 16:00 Wechaty Plugin开发者 王墨炱 基于插件的Wechaty开发](https://www.bilibili.com/video/BV1sz4y1973D)
```

**Comment by huan** [comment]
I fully agree with @lijiarui about the suggestion of adding slide links from our speakers.

And also, we have saved lots of the screenshot photos in our WeChat room, which is mainly for use in this blog post: please add those slide screenshots for each speaker, so that we can improve the readability for our readers.

**Comment by rickyyin98** [comment]
Plugin Blog all fixed:
- add speakers slides or live code link
- change the agenda to a better display
- add more screenshoot

@huan @lijiarui

**Comment by rickyyin98** [comment]
Puppet Blog Done

@huan @lijiarui

**Review by huan — APPROVED** [review]
LGTM

**Comment by huan** [comment]
Thank you very much for your two great blog posts, it's very valuable to the community!

**Pull request merged**